tm2py/controller.py

```diff
@@ -310,7 +310,8 @@ def _queue_components(self, run_components: Collection[str] = None):
             _final_components = [c for c in _final_components if c in run_components]
         
             #printing run components by iteration type (initial, global, final) in order
-            print('Initial components', _initial_components)
+            if self.config.run.start_iteration == 0:
```

Comment:
I'm not in favor of adding this condition. @vivekyadav26

Reply:
Sorry, I am missing the background with this change. Jayne made these changes before she left. Can you suggest the changes ?

Reply:
I suggest reverting this change.

Reply:
reverted the changes.